### ISSUE TITLE
Moodle Questionnaire Plugin - Accessibility issue

### ISSUE THREAD

**Issue by kerryvanucl** [remotelearner/moodle-mod_questionnaire#633]
We found an accessibility issue with Moodle Questionnaire.  

Currently, when a required field is left blank, it is not clear to a screen reader user why they can't proceed to the next page or that they have made an error. A notification is read out that they need to complete the field but we recommend providing a little more detail in the description so a screen reader user understands that they cannot proceed because there is an error because they missed completing a field.

Many thanks :)

https://github.com/user-attachments/assets/174c8f69-c968-423f-ba0d-f1af2a4b4571
